listener_test.go

```diff
@@ -109,9 +110,15 @@ func createNewListenerWithConnectionString() (*Listener, error) {
 	return NewListener(ListenerWithConnectionString(connStr))
 }
 
-func createTestHandler() Handle {
-	return func(ctx context.Context, message string, messageType string) error {
-		return nil
+func createTestHandler() handle.Handle {
+	return func(ctx context.Context, message string, messageType string) handle.Handler {
+		return handle.Complete()
+	}
+}
```

Comment:
here is a handler with the new syntax.
call handle.Complete() instead of return nil. although, return nil is still accepted as success.
call handle.Abandon() or handle.Error(err) to fail the handling and leave the message on the queue